tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>Here I'm getting an error in colab:</p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
&lt;ipython-input-20-01035994e382&gt; in &lt;module&gt;()
      1 clustered_tflite_file = '/tmp/clustered_mnist.tflite'
      2 converter = tf.lite.TFLiteConverter.from_keras_model(final_model)
----&gt; 3 tflite_clustered_model = converter.convert()
      4 with open(clustered_tflite_file, 'wb') as f:
      5   f.write(tflite_clustered_model)


3 frames
/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/convert_to_constants.py in _get_tensor_data(func)
    215       data = map_index_to_variable[idx].numpy()
    216     else:
--&gt; 217       data = val_tensor.numpy()
    218     tensor_data[tensor_name] = {
    219         "data": data,

AttributeError: 'Tensor' object has no attribute 'numpy'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
@Ruomei: you can install the new package at 

"pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple tensorflow-model-optimization==0.4.0.dev2"

For testing.